dashboard/test/controllers/plc/evaluation_questions_controller_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
+
+class Plc::EvaluationQuestionsControllerTest < ActionController::TestCase
+  setup do
+    user = create (:admin)
+    sign_in(user)
+    course = create(:plc_course)
+    @course_unit = create(:plc_course_unit, plc_course: course)
+    @plc_evaluation_question = create(:plc_evaluation_question, plc_course_unit: @course_unit)
+  end
+
+  test "should destroy plc_evaluation_question" do
+    assert_difference('Plc::EvaluationQuestion.count', -1) do
```

Comment:
A better test might be to make sure this specific `EvaluationQuestion` was destroyed, not just that the count decreased.